requirements-python.txt

```diff
@@ -29,7 +29,9 @@ selenium==3.141.0
 mysql-connector-python==8.0.19
 docker==4.2.0
 python-jenkins==1.4.0
-ssh2-python==0.18.0.post1
+# ssh2-python==0.18.0.post1
+# fix segmentation fault with ssh2-python
+https://github.com/dkropachev/ssh2-python/archive/fix_segmentation.zip
```

Comment:
your suggest fix wasn't merged to the newer versions released ?

Reply:
I see you did, 
https://github.com/ParallelSSH/ssh2-python/pull/118#issuecomment-707710718

but seem like you missed some procedure, or pointed it to the wrong repo...

Reply:
@fruch the issue is that they need a test that 100% reproduces the issue and we don't have one since we will need to invest a lot of time in it (C code) so currently we decided not to contribute and do it in the future Sprints. @dkropachev did you open a Trello task for this?